components/ActionSheet/tests/unit/Theming/MDCActionSheetThemingTest.m

```diff
@@ -63,10 +67,17 @@ - (void)testActionSheetThemingTest {
   // Given
   MDCTypographyScheme *typographyScheme =
       [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201804];
+  MDCActionSheetAction *fakeActionOne = [MDCActionSheetAction actionWithTitle:@"Action 1"
```

Comment:
Looks like some functional changes got mixed-in with the Beta-to-Ready changes. Could you please revert these or split them into a separate PR?

Reply:
This is addressed in #7183 which I will merge before this lands.